### PR TITLE
Interactivity

### PR DESCRIPTION
**Pull request by FionaLippert**

### CODE REVIEW

**Comment by FionaLippert** [comment]
Please review. I can do the conflict resolving before merging if you have any difficulties with that.

File: src/Fiona/run_classroom_seating.py [hunk 1]
```diff
@@ -4,82 +4,71 @@
...
     utilities: list of booleans [location, sociability, friendship] specifying which utilities are used in the model
 
 Returns:
-    blocking: 2D array representing the classroom.
+    image: 2D array representing the classroom.
                 - The level of seat blocking is represented by decreasing negative values (the smaller the costlier to reach the seat)
                 - The level of "happiness" of each student is represented by positive values (the larger the happier)
+    info: 3D array comprising the 4 matrices 'seat_utilities', 'student_IDs', 'student_sociabilities' and 'initial_happiness'
 """
-def get_seat_blocking(model, utilities):
```
Comment (jerbaroo) [inline src/Fiona/run_classroom_seating.py]: I tried running via `python3 run_classroom_seating.py generate` and got the following:

```
  File "run_classroom_seating.py", line 119, in <module>
    image, info = get_seat_blocking(models[i], model_utilities[i])
NameError: name 'get_seat_blocking' is not defined
```
@FionaLippert

Comment (FionaLippert) [reply src/Fiona/run_classroom_seating.py]: Sorry, fixed it.

**Comment by jerbaroo** [comment]
Looks great @FionaLippert 🎉 

yeah if you can resolve and merge please, there are some changes i'm not sure about what to keep

**Pull request merged**